astropy/units/astrophys.py

```diff
@@ -203,4 +163,4 @@
 # standard units defined here.
 from .utils import generate_unit_summary as _generate_unit_summary
 if __doc__ is not None:
-    __doc__ += _generate_unit_summary(globals())
+    __doc__ += _generate_unit_summary(globals())
```

Comment:
You'll need a similar stanza at the end of your new `misc.py` as well, including the `CLEANUP` part above. And you need to import from `misc.py` in `__init__.py` (just add everywhere `astrophys` is done). Finally, also add a line in `docs/units/index.rst` (again look for `astrophys`).

Please also run tests locally before pushing - all tests are failing now since the units are not imported properly. Please do include the pep8 tests - you have many missing CR at the end of files (see instructions for astropy development).